src/cose/decoder.rs

```diff
@@ -44,42 +44,42 @@ fn get_map_value(map: &CborType, key: &CborType) -> Result<CborType, CoseError>
     }
 }
 
-/// COSE_Sign = [
+/// `COSE_Sign` = [
```

Comment:
This is a little odd - maybe we should make this a non-documentation comment (i.e. '//' instead of '///')? Not a big deal, though.

Reply:
Yeah, I wasn't quite sure how we want to handle this. I think I'll leave as is in this PR for now. We don't build the docs yet and have to  polish documentation at some point anyway.